src/components/headline/styled.ts

```diff
@@ -22,6 +22,10 @@ const Title = styled.h1`
   text-overflow: ellipsis;
 `;
 
-const SubTitle = styled.h2``;
+const SubTitle = styled.h2`
+  & > strong {
+    margin-left: 5px;
+  }
```

Comment:
😢

Reply:
I think it's better to use {' '} in the code rather than setting random margin to the strong element.

Reply:
Oke